scripts/gufi_stats

```diff
@@ -1056,6 +1059,18 @@ def filesize_log2_bins(_config, args, _where):
 def filesize_log1024_bins(_config, args, _where):
     return size_bins(args, 1024, 'f')
 
+def direntries_log2_bins(_config, args, _where):
+    return size_bins(args, 2, 'd', 'totfiles')
+
+def direntries_log1024_bins(_config, args, _where):
+    return size_bins(args, 1024, 'd', 'totfiles')
```

Comment:
maybe rename to `dirfilecount_*` since you are pulling `totfiles` and not `totfiles + totlinks`

Reply:
Can add multiple functions:
1. dirfilecount for just files
2. dirlinkcount for just links
Cannot do direntrycount bec need totsubdirs. I'll submit an Issue asking for totsubdirs.